### DISCUSSION TITLE
Support for UTMStack Integration with Cisco Switch

### DISCUSSION THREAD

**Discussion by Jeffzieee** [utmstack/UTMStack#688] [Q&A] [answered]
Could you please provide assistance on setting up integration of Cisco Switch to the SIEM? Also, I am not quite sure what it means by Windows and Linux OS on configuring UTMStack agent on a Cisco Switch. Because there is no Windows or Linux OS for Cisco. 
This is applicable to all integrations. 
![cisco-switch-integration](https://github.com/utmstack/UTMStack/assets/68960919/dd269e91-d895-499e-b587-32fcb7748eb3)

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @Jeffzieee, the CISCO SWITCH integration needs to install, the linux agent or windows agent integration first, according to the OS where you are installing the agent. So, as the guide says, you must configure your cisco switch to send the logs through syslog to those ports, then install an agent, and, on the agent's machine, do the 2 and 3 steps. The platform in step 2 refers to the agent's machine (windows or supported linux). After selecting values (step 2) you will see a command to execute on the agent's machine that will enable the agent's module.
Best regards
